ssl/t1_enc.c

```diff
@@ -410,6 +451,7 @@ int tls1_change_cipher_state(SSL *s, int which)
         goto err;
     }
 
+# ifndef __FreeBSD__
```

Comment:
Should this perhaps be `#ifdef __linux` or `#ifdef OPENSSL_SYS_LINUX` instead of `#ifndef __FreeBSD__`?  If so, some of the `#else` in this patch should also be replaced with `#endif / #ifdef __linux` as well?

Reply:
While we only have two platforms that have this capability I'm ok with just keeping this as you have it here. If more platforms get added then we can revisit then.

Reply:
I would prefer that you mention Linux here at least in a comment if nothing else.